packages/flutter_tools/templates/create/.travis.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+os:
+  - osx
+osx_image: xcode7.2
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git /tmp/flutter
```

Comment:
I've done that before!  :)

> Agreed. Though, this is only a template. After the initial flutter create, the developer is in full (and sole) control of the repo and the branch.

That's true, but the starter app template should show all the best practices.  It's going to end up being copy/pasted everywhere.

> Should I modify this to default to alpha instead?

That's probably an improvement, but doesn't solve the underlying problem.  You need to test your app against the same version of Flutter that you're developing your app against.  Whatever branch we pick will be a moving target and not synchronized with whatever the developer has on their machine.

> Since putting the Flutter repo next to the project is only a recommendation (one our users may choose to ignore), I decided to setup the test harness so that it downloads the Flutter repo at a known location and unconditionally rewrite the path dependency.

That won't always work.  Consider the case of an app that depends on package X and package X depends on flutter being at `../flutter/packages/flutter`.  You've re-written the path for the app itself but you haven't change package X.  That means the `pub get` step will fail because there will be conflicting requirements for where to get the flutter package.

Another example: suppose the developer also has a dependency on cassowary.  The rewrite will also miss that and cause a conflict.

I think we need to solve this problem by having the starter app recommend a technology for expressing its dependency on flutter.git.  The bootstrapping is slightly odd because you already have flutter.git when you run "create".  We might need to change how we bootstrap to be something more like `curl https://... | bash`